spec/components/orangelight/advanced_search_form_component_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Orangelight::AdvancedSearchFormComponent, type: :component, advanced_search: true do
+  subject(:render) do
+    component.render_in(view_context)
+  end
+  let(:component) { described_class.new(url: '/whatever', response:, params:) }
+
+  let(:response) do
+    Blacklight::Solr::Response.new({}.with_indifferent_access, {})
+    # Blacklight::Solr::Response.new({ facet_counts: { facet_fields: {
```

Comment:
Could you please remove the commented-out response?

Reply:
Removed